src/mapblock_mesh.cpp

```diff
@@ -1234,6 +1235,11 @@ MapBlockMesh::MapBlockMesh(MeshMakeData *data, v3s16 camera_offset):
 
 	translateMesh(m_mesh, intToFloat(data->m_blockpos * MAP_BLOCKSIZE - camera_offset, BS));
 
+	if (m_enable_shaders) {
```

Comment:
Is it necessary to always do this if shaders are enabled?  I was under the impression that tangents only needed to be calculated when parallax occlusion is enabled.